x/protocolpool/keeper/keeper.go

```diff
@@ -201,17 +205,23 @@ func (k Keeper) SetToDistribute(ctx context.Context, amount sdk.Coins, addr stri
 		return err
 	}
 
-	totalStreamFundsPercentage := math.ZeroInt()
-	err = k.RecipientFundPercentage.Walk(ctx, nil, func(key sdk.AccAddress, value math.Int) (stop bool, err error) {
-		totalStreamFundsPercentage = totalStreamFundsPercentage.Add(value)
-		return false, nil
-	})
-	if totalStreamFundsPercentage.GT(math.NewInt(100)) {
-		return fmt.Errorf("total funds percentage cannot exceed 100")
-	}
+	var totalStreamFundsPercentage math.Int
+	hasTotalFund, err := k.TotalFundPercentage.Has(ctx)
```

Comment:
Just realized we can't keep the total fund percentage, given that eventually some of them will expire

Reply:
reverted